main/src/org/destinationsol/game/planet/PlanetManager.java

```diff
@@ -60,10 +60,10 @@ public PlanetManager(TextureManager textureManager, HullConfigManager hullConfig
         mySysConfigs = new SysConfigs(textureManager, hullConfigs, itemManager);
         myMazeConfigs = new MazeConfigs(textureManager, hullConfigs, itemManager);
 
-        mySystems = new ArrayList<SolSystem>();
-        myMazes = new ArrayList<Maze>();
-        myPlanets = new ArrayList<Planet>();
-        myBelts = new ArrayList<SystemBelt>();
+        mySystems = new ArrayList<>();
```

Comment:
Not necessarily complaining, but why does the type detail go away here? For something like `Planet planet : myPlanets` later wouldn't it complain about generics/type somehow?